e2e/integration/utils/context.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package utils
+
+import (
+	"net/url"
+	"sort"
+	"strings"
+)
+
+//Test context struct
+type TestContext struct {
+	Cfg Config
+}
+
+//function to get the testcontext Object.
+func NewTestContext(cfg Config) *TestContext {
+	return &TestContext{
+		Cfg: cfg,
+	}
+}
+
+func MapLables(ls map[string]string) string {
```

Comment:
Pls add a comment with a short description above this function explaining its functionality

Reply:
nit: MapLables -> MapLabels ??